src/js/controller/LayersListController.js

```diff
@@ -46,26 +49,26 @@
 
   ns.LayersListController.prototype.createButtonTooltips_ = function () {
     var addTooltip = pskl.utils.TooltipFormatter.format('Create a layer', null, [
-      {key : 'shift', description : 'Clone current layer'}
+      { key: 'shift', description: 'Clone current layer' }
```

Comment:
Most of the codebase has a space before the colon in objects. 
It would be nice to preserve this in the patch, especially on lines which were not touched by the patch otherwise.